libs/stepper/package.json

```diff
@@ -20,7 +20,9 @@
   "peerDependencies": {
     "@angular/common": "^9.0.0",
```

Comment:
Why here are `@angular` modules version 9 ?

Reply:
Package updated to v10.